sonar-python-plugin/src/main/java/org/sonar/plugins/python/pylint/PylintRuleRepository.java

```diff
@@ -19,32 +19,232 @@
+    map.put("E0001", "5min");
+    map.put("E0011", null);
+    map.put("E0012", null);
+    map.put("E0100", "30min");
+    map.put("E0101", "5min");
+    map.put("E0102", "15min");
+    map.put("E0103", "15min");
+    map.put("E0104", "5min");
+    map.put("E0105", "10min");
+    map.put("E0106", "15min");
+    map.put("E0107", "3min");
+    map.put("E0108", "5min");
+    map.put("E0109", "10min");
+    map.put("E0111", "10min");
+    map.put("E0202", "15min");
+    map.put("E0203", "15min");
+    map.put("E0211", "10min");
+    map.put("E0213", "5min");
+    map.put("E0221", "20min");
+    map.put("E0222", "30min");
+    map.put("E0235", "10min");
+    map.put("E0236", "10min");
+    map.put("E0238", "20min");
+    map.put("E0501", "5min");
+    map.put("E0502", "5min");
+    map.put("E0503", "10min");
+    map.put("E0601", "15min");
+    map.put("E0602", "10min");
+    map.put("E0603", "3min");
+    map.put("E0604", "3min");
+    map.put("E0611", "10min");
+    map.put("E0701", "30min");
+    map.put("E0702", "20min");
+    map.put("E0710", "15min");
+    map.put("E0711", "5min");
+    map.put("E0712", "20min");
+    map.put("E1001", "15min");
+    map.put("E1002", "15min");
+    map.put("E1003", "10min");
+    map.put("E1004", "10min");
+    map.put("E1101", "10min");
+    map.put("E1102", "3min");
+    map.put("E1103", "10min");
+    map.put("E1111", "20min");
+    map.put("E1120", "5min");
+    map.put("E1121", "5min");
+    map.put("E1122", "3min");
+    map.put("E1123", "3min");
+    map.put("E1124", "5min");
+    map.put("E1200", "10min");
+    map.put("E1201", "10min");
+    map.put("E1205", "10min");
+    map.put("E1206", "10min");
+    map.put("E1301", "3min");
+    map.put("E1302", "5min");
+    map.put("E1303", "10min");
+    map.put("E1304", "10min");
+    map.put("E1305", "5min");
+    map.put("E1306", "5min");
+    map.put("E1310", "10min");
+    map.put("E9900", "3min");
+    map.put("F0001", null);
+    map.put("F0002", null);
+    map.put("F0003", null);
+    map.put("F0004", null);
+    map.put("F0010", null);
+    map.put("F0202", null);
+    map.put("F0220", null);
+    map.put("F0321", null);
+    map.put("F0401", null);
+    map.put("I0001", null);
+    map.put("I0010", null);
+    map.put("I0011", null);
+    map.put("I0012", null);
+    map.put("I0013", null);
+    map.put("R0201", "20min");
+    map.put("R0401", "2h");
+    map.put("R0801", null);
+    map.put("R0901", "3h");
+    map.put("R0902", "2h");
+    map.put("R0903", "1h");
+    map.put("R0904", "2h");
+    map.put("R0911", "20min");
+    map.put("R0912", null);
+    map.put("R0913", "30min");
+    map.put("R0914", "20min");
+    map.put("R0915", null);
+    map.put("R0921", "20min");
+    map.put("R0922", "45min");
+    map.put("R0923", "20min");
+    map.put("W0101", "15min");
+    map.put("W0102", "15min");
+    map.put("W0104", "15min");
+    map.put("W0105", "15min");
+    map.put("W0106", "15min");
+    map.put("W0106", "15min");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Verify this is the key that was intended; it was already set before. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS4143)